package.json

```
@@ -7,52 +7,48 @@
-		"lodash": "^4.17.21",
 		"newspack-components": "^2.2.1",
-		"react": "^17.0.2",
-		"redux": "^4.2.1",
+		"redux": "^5.0.0",
 		"redux-saga": "^1.3.0",
 		"regenerator-runtime": "^0.14.1",
 		"swiper": "11.1.4"
 	},
 	"scripts": {
 		"cm": "newspack-scripts commit",
 		"semantic-release": "newspack-scripts release --files=newspack-blocks.php",
-		"build": "newspack-scripts build",
-		"start": "npm ci && newspack-scripts watch",
-		"watch": "newspack-scripts watch",
+		"build": "newspack-scripts wp-scripts build",
+		"start": "npm ci && npm run watch",
+		"watch": "newspack-scripts wp-scripts start",
 		"test": "newspack-scripts test",
 		"lint": "npm run lint:scss && npm run lint:js",
-		"lint:js": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
-		"lint:js:staged": "eslint --ext .js,.jsx,.ts,.tsx",
+		"lint:js": "newspack-scripts wp-scripts lint-js '**/{src,includes}/**/*.{js,jsx,ts,tsx}'",
+		"lint:js:staged": "newspack-scripts wp-scripts lint-js --ext .js,.jsx,.ts,.tsx",
+		"format:js": "newspack-scripts wp-scripts format '**/{src,includes}/**/*.{js,jsx,ts,tsx}'",
 		"lint:php:staged": "./vendor/bin/phpcs --filter=GitStaged",
-		"format:js": "prettier 'src/**/*.{js,jsx,ts,tsx}' --write",
-		"lint:scss": "stylelint '**/*.scss' --customSyntax postcss-scss --config=./node_modules/newspack-scripts/config/stylelint.config.js",
-		"format:scss": "prettier --write 'src/**/*.scss'",
-		"lint:scss:staged": "stylelint --customSyntax postcss-scss --config=./node_modules/newspack-scripts/config/stylelint.config.js",
+		"lint:scss": "newspack-scripts wp-scripts lint-style '**/{src,includes}/**/*.scss' --customSyntax postcss-scss",
+		"lint:scss:staged": "newspack-scripts wp-scripts lint-style --customSyntax postcss-scss",
+		"format:scss": "newspack-scripts wp-scripts lint-style '**/{src,includes}/**/*.scss' --customSyntax postcss-scss --fix",
 		"i18n": "NODE_ENV=production npm run build; bash ./bin/update-translations.sh",
 		"typescript:check": "newspack-scripts typescript-check",
```

Getting an error when I run this:

```
> @automattic/newspack-blocks@3.3.0 typescript:check
> newspack-scripts typescript-check

[newspack-scripts@1.0.0] Starting TypeScript check…
src/blocks/donate/edit/index.tsx:11:22 - error TS2307: Cannot find module '@wordpress/api-fetch' or its corresponding type declarations.

11 import apiFetch from '@wordpress/api-fetch';
                        ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/blocks/donate/edit/index.tsx:11

npm verbose exit 2
npm verbose code 2
```

---

That might be due to the [dependency extraction](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dependency-extraction-webpack-plugin/) for `@wordpress/*` modules. Maybe we can just ignore errors like that for now.

---

Weird! I'm not seeing this error when I run this locally.

Try running `rm -rf node_modules && start` from the Blocks repo directory and see if it resolves things? Sometimes Node has a habit of messing up the install if there are already `node_modules` installed from a prior run...

---

This should be fixed now with the `v5.6.0-alpha.4` version (and up) of this package.

---

> This should be fixed now with the v5.6.0-alpha.4 version (and up) of newspack-scripts.

Yup! Seems to be all good now 👍